easybuild/easyconfigs/f/FALL3D/FALL3D-9.0.1-foss-2023a.eb

```
@@ -0,0 +1,40 @@
+easyblock = 'CMakeMakeCp'
+name = 'FALL3D'
+version = '9.0.1'
+
+homepage = 'https://gitlab.com/fall3d-suite/fall3d'
+description = "FALL3D is an open-source volcanic ash dispersal model."
+
+toolchain = {'name': 'foss', 'version': '2023a'}
```

This doesn't seem to depend on BLAS/FFTW/ScaLAPACK, so I think we can use `gompi`?
```suggestion
toolchain = {'name': 'gompi', 'version': '2023a'}
```